source/modulo_components/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>modulo_components</name>
-  <version>5.2.0</version>
+  <version>5.2.1</version>
   <description>Modulo base classes that wrap ROS2 Nodes as modular components for the AICA application framework</description>
```

Comment:
```suggestion
  <description>Modulo base classes that wrap ROS 2 Nodes as modular components for the AICA application framework</description>
```
since we are here :yum: